config/jobs/kubernetes/sig-testing/kind.yaml

```diff
@@ -33,8 +33,11 @@ periodics:
         name: cgroup
       resources:
         requests:
-          # We can adjust this after we know the requirements.
-          memory: "6Gi"
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
```

Comment:
I sampled a running job with `kubectl top po -n=test-pods $pod` to get a better idea for these. I also took a look at the resource usage on the node with some other tools (IE `top` mainly) to sanity check.